modules/ROOT/pages/cassandra/cassandra-use-cases.adoc

```diff
@@ -0,0 +1,231 @@
+= Use Cases - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Example: Create a Cassandra Keyspace using Studio
+
+Create a Cassandra keyspace which groups column families, then create a Cassandra table.
```

Comment:
If my goal is to use Cassandra connector to integrate a Cassandra database with some other data store--what part of that process is this activity?

Reply:
Fixed, I think.